src/styles/blocks/card.scss

```diff
@@ -0,0 +1,105 @@
+@import "../utils/variables";
+
+.card {
+  width: $width-card;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid $border-color-card;
+  transition: all 300ms ease-in-out;
```

Comment:
Try to set it for specific parameters, not for all.

Reply:
Not fixed

Reply:
Don't use transition all, specify to what properties it should apply.
![image](https://github.com/mate-academy/layout_catalog/assets/63243136/3a10d0fb-5cd1-4c80-a8d5-a54b60da834f)
If you don't get what you need to fix, ask mentors.